app/app.go

```diff
@@ -418,6 +429,15 @@ func (app *XplaApp) BlockedModuleAccountAddrs(modAccAddrs map[string]bool) map[s
 	// remove module accounts that are ALLOWED to received funds
 	delete(modAccAddrs, authtypes.NewModuleAddress(govtypes.ModuleName).String())
 
+	blockedPrecompilesHex := evmtypes.AvailableStaticPrecompiles
+	for _, addr := range vm.PrecompiledAddressesBerlin {
```

Comment:
We need to add PrecompiledContract Address defined in xpla. BlockedModuleAccountAddrs is executed before RegistPrecompiledContract.

Reply:
[d30da4d](https://github.com/xpladev/xpla/pull/171/commits/d30da4dbdcf31a972bf0390f39fbd7acb9ddf619)